codec-http2/src/main/java/io/netty/handler/codec/http2/InboundHttp2ToHttpAdapter.java

```diff
@@ -55,7 +54,7 @@ public boolean mustSendImmediately(FullHttpMessage msg) {
         @Override
         public FullHttpMessage copyIfNeeded(FullHttpMessage msg) {
             if (msg instanceof FullHttpRequest) {
-                FullHttpRequest copy = ((FullHttpRequest) msg).replace(Unpooled.buffer(0));
+                FullHttpRequest copy = ((FullHttpRequest) msg).replace(ByteBufAllocator.DEFAULT.buffer(0));
```

Comment:
maybe it would be better to use:

```
FullHttpRequest req = (FullHttpRequest) msg;
FullHttpRequest copy = req.replace(req.content().alloc().buffer(0)));
```

Reply:
Agree. Fixed.